packages/next/server/image-config.ts

```diff
@@ -33,3 +36,7 @@ export const imageConfigDefault: ImageConfigComplete = {
   minimumCacheTTL: 60,
   formats: ['image/webp'],
 }
+
+export const getImageConfigDefault: () => ImageConfigComplete = () => ({
+  ...imageConfigDefault,
+})
```

Comment:
Lets just put it in this function and remove the variable above?

```suggestion
export const getImageConfigDefault: () => ImageConfigComplete = () => ({
  deviceSizes: [640, 750, 828, 1080, 1200, 1920, 2048, 3840],
  imageSizes: [16, 32, 48, 64, 96, 128, 256, 384],
  path: '/_next/image',
  loader: 'default',
  domains: [],
  disableStaticImages: false,
  minimumCacheTTL: 60,
  formats: ['image/webp'],
})
```

Reply:
Wouldn't that be a breaking change? The approach I took would be backwards compatible.

Totally willing to make the update but wanted to call that out.

Reply:
I'd suggest using the backward compatible approach so that it can be patched, and look to remove the variable in the next major release in order to follow semver.

Reply:
This file is not exported to users and is not documented so its not covered by semver.

In fact, the same with the original issue: `loadConfig()` is internal and not covered by semver.

Reply:
Yes, but you do have consumers using it, namely, `@nrwl/next`

Reply:
At 50,653 weekly downloads I think it's worth supporting the patch.

Reply:
PR with objects removed: https://github.com/vercel/next.js/pull/32477